### PR TITLE
[DF-16] Register

### PR DESCRIPTION
**Pull request by octobrekids**
In this branch, including by
- Register Page
- Register Page Test
- Register Page Storybook

### CODE REVIEW

File: src/common/tests/pages/__snapshots__/register.spec.tsx.snap [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Pages/Register test Render correctly 1`] = `<RegisterPage />`;
```
Comment (yee2542) [inline src/common/tests/pages/__snapshots__/register.spec.tsx.snap:3]: by looking this is no guarantee what a change of inside component, so use `mount` for snapshots

File: src/common/tests/pages/register.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import FormCenterLayout from 'Modules/user/components/Layouts/FormCenter';
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import RegisterPage from 'Modules/user/pages/register';
+
+describe('Pages/Register test', () => {
+  it('Render correctly', () => {
+    const wrap = <RegisterPage />;
+    expect(wrap).toMatchSnapshot();
```
Comment (yee2542) [inline src/common/tests/pages/register.spec.tsx:9]: here too

File: src/common/tests/pages/login.spec.tsx [hunk 3]
```diff
@@ -10,14 +10,14 @@ describe('Pages/Login test', () => {
     expect(wrap).toMatchSnapshot();
```
Comment (yee2542) [inline src/common/tests/pages/login.spec.tsx:10]: you have to mount it to match a snapshots

**Review by yee2542 — CHANGES_REQUESTED** [review]
please check

File: src/common/tests/pages/__snapshots__/register.spec.tsx.snap (hunk 1 above)
Comment (octobrekids) [reply src/common/tests/pages/__snapshots__/register.spec.tsx.snap]: fix here 2b36b8af941a2ceb43da8923b6e881c0737102bb

File: src/common/tests/pages/register.spec.tsx (hunk 2 above)
Comment (octobrekids) [reply src/common/tests/pages/register.spec.tsx]: fix here too 2b36b8af941a2ceb43da8923b6e881c0737102bb

File: src/common/tests/pages/login.spec.tsx (hunk 3 above)
Comment (octobrekids) [reply src/common/tests/pages/login.spec.tsx:10]: fix here too eb52e9cf7ef6a4c6d91c185facf56653e8c3bc4b

File: src/modules/user/pages/register/index.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+              <P>
+                Already have an account?
+                <Link href="/login">
+                  <a> Log in</a>
+                </Link>
+              </P>
+            </Form.Item>
+          </Form>
+        </Col>
+      </Row>
```
Comment (yee2542) [inline src/modules/user/pages/register/index.tsx:104]: ยกออก

**Review by yee2542 — CHANGES_REQUESTED** [review]

File: src/modules/user/pages/register/index.tsx (hunk 4 above)
Comment (octobrekids) [reply src/modules/user/pages/register/index.tsx]: ยกแล้ว 3ced1d866a4c84bf7d6b2b102caa2d5f9550a954

**Pull request merged**